plonky2/src/util/serialization/mod.rs

```diff
@@ -1689,12 +1697,14 @@ pub trait Write {
         &mut self,
         gate: &GateRef<F, D>,
         gate_serializer: &dyn GateSerializer<F, D>,
+        common_data: &CommonCircuitData<F, D>,
```

Comment:
Same with this

Reply:
common_data is actually being used in quite a lot of places (and consistently across this file). Do you want me to address this particular case still?

Reply:
Hm maybe it should just be called `common_data` everywhere (instead of `cd`)?

Reply:
Ok! Addressed in additional commit.